### ISSUE TITLE
required owned and context roles

### ISSUE THREAD

**Issue by rdeltour** [w3c/dpub-aria#15]
It seems there's something wonky with the relationships established with _required owned elements_, _required context role_, and role subclasses defined in DPUB ARIA.

ARIA 1.1 has a note in the [Required own elements](https://www.w3.org/TR/wai-aria-1.1/#mustContain) section that says:

> An element with a subclass role of the 'required owned element' does not fulfill this requirement. For example, the `list` role requires ownership of an element using either the `listitem` or `group` role. Although the `group` role is the superclass of `row`, adding a owned element with a role of `row` will not fulfill the requirement that list must own a `listitem` or a `group`.

This means for instance that `doc-biblioentry` cannot be used as the unique children of a `list`, which invalidates Example 6:

```html
<section role="doc-bibliography">
   <h1>Cited Works</h1>
   <div role="list">
      <p role="doc-biblioentry" id="b8cab5dd-bc24-459c-9858-7afa9da69b64">
         John Steinbeck, The Grapes of Wrath (New York: The Viking Press, 1939)
      </p>
   </div>
   …
</section>
```

In other words, `doc-bibliography` is only usable on `li` children of `ol` and `ul`.

Can we clarify that with the ARIA folks (cc @w3c/aria)? This looks like an oversight in how DPUB extends ARIA, or maybe even in ARIA's own extensibility.

**Comment by mattgarrish** [comment]
> In other words, `doc-bibliography` is only usable on `li` children of `ol` and `ul`.

But it wouldn't be, if we follow that logic, would it? Both of those have the implicit role of `list` so they also couldn't allow listitem subclasses. Strictly interpreting that note, you always have to create new list types in order to create subclasses of listitem, at least outside the core.

I agree that the note adds confusion, especially since it's technically only informative and I don't see the rule expressed normatively, but it can't be solved here. I appreciate why it exists, too, but the problem is that extension modules can't modify the core module, and the core module doesn't reference extension roles, so until a role is added to the core there will be this ambiguity.

What we probably need is a way to confirm that the roles are, indeed, expected to be included in the expected owned role list when in extensions ("required owner", or something like that).

But that assumes that the way we created the vocabulary is correct, which I'm holding out hope we did. If not, then the problem will require reworking the vocabulary.

/cc @TzviyaSiegman @joanmarie

**Comment by rdeltour** [comment]
See also w3c/aria#748 which proposes a stricter definition of "owned", which would break the multi-ancestors required contexts of roles like `doc-biblioentry` (which requires both a direct `list` owner and `doc-bibliography` grand-owner).
